README.md

```diff
@@ -116,32 +116,76 @@ On OS X:
     sudo ln -s /Library/Internet\ Plug-Ins/JavaAppletPlugin.plugin/Contents/Home/bin/java /usr/bin/java
     ```
 
-**Download & setup Solr**
+**Download Solr**
 
-``` bash
-wget http://mirror.sdunix.com/apache/lucene/solr/4.10.4/solr-4.10.4.tgz
-tar -xvf solr-4.10.4.tgz
-sudo cp -R solr-4.10.4/example /opt/solr
+NYC Councilmatic uses Solr version 7 to power search fro legislation. If you already have Solr version 7 installed for another app, then you can by-pass these installation instructions. Otherwise, carefully read on.
+
+Download latest solr distribution from a [reliable mirror](http://www.apache.org/dyn/closer.cgi/lucene/solr/). The Apache foundation suggests using the [University of Toronto mirror](http://mirror.dsrg.utoronto.ca/apache/lucene/solr/). You need to download two files:
```

Comment:
I think they purposely only host the signature files on the [main apache server](http://archive.apache.org/dist/lucene/solr/7.1.0/) so that there is only one, central place to use to verify that the files are what you expect them to be.